AlpheusCore/ExecuteCommand.fs

```diff
@@ -14,32 +14,58 @@ let internal annotateLine (methodId:string) (channel:string) (line:string) =
     let name = sprintf "%s [%s]" methodId channel
     formatLine name line
 
+/// Executes program as a separate process, capturing stderr and stdout, processing them with print function callback.
+/// Also supplies the output with outputAnnotationId label
+/// workingDir is both the path for looking for the program and workingDir for the new process to be executed
+let runCmdLocallyAsync print outputAnnotationId program args (workingDir:string) =
+    if not(Path.IsPathRooted(workingDir)) then
+        invalidArg "workingDir" "working directory must be specified as full path"
+    async {
+        let printStream (annotate:string->string) (stream:StreamReader) = 
+            async {
+                do! Async.SwitchToNewThread()
+                while not stream.EndOfStream do
+                    let! line = Async.AwaitTask(stream.ReadLineAsync())
+                    line |> annotate |> print
+            }
+        
+        use p = new System.Diagnostics.Process()
+        // There is tricky situation with command (see github issue #1)
+        // sometimes the user specifies "command" as an path to executable relative to the "working directory"
+        // that is the prioritized behavior
+        // if the executable is not found we try to run the "command" searching it in PATH
+        let program = 
+            let absExecutableCandidate = Path.Combine(workingDir,program)
```

Comment:
Why do we need an absolute path? Is there a problem with a path to the executable relative to `Process.StartInfo.WorkingDirectory`?